charters/charter-template.md

```diff
@@ -0,0 +1,30 @@
+
+# Project Name
+
+
+## Description
+
+## Objective
+
+## In-scope
+
+## Out-of-scope
+
+## Milestones
+
+## Roles
+
+### Project Lead
```

Comment:
As stated over email, @TimothyTickle and my vote here is for coordinator, product owner, and technical lead to be the roles.

Reply:
Tim proposed _Coordinator_ on an email thread with the current leads. At this point, the response is a preference to maintain _Lead_ from the DCP Quarterly goals in favor of _Project Lead_.  My approach is to allow the current membership of each role (lead, product owner, and technical lead) to reach consensus on their own titles and responsibilities.

Reply:
I would like us to avoid using the term "product owner" as it is ambiguous and misleading. Let's use the next best acceptable alternative.

Reply:
@kislyuk Do you have a suggestion on another alternative? It is a pretty defined role for someone who is owner of the product. In reality, all role names are ambiguous and depend on the institution using them and the person in them. Technical Lead and Project Lead are just as ambiguous.

If we were to have definitions of the role somewhere would that satisfy the requirement?

Reply:
@kislyuk - It's not ambiguous. It may be misleading. It's a defined role specific to **Scrum** and there's a [historical reason](https://www.romanpichler.com/blog/product-manager-vs-product-owner/) that it was introduced over product manager at the time. It does imply that DCP is demanding or assuming adherence to a methodology.

Reply:
I think it is ambiguous and misleading.

I think the terms "project manager", "product manager", and "technical lead" are less ambiguous.

I confess I was not actually able to follow the reasoning in your link at all, @brianraymor.

Reply:
I think "product owner", "project manager", "product manager", and "technical lead" are not interchangeable and each have their own specific meaning (that may differ based on who you talk to).  @brianraymor I think whatever term we use here (or terms) it's best to define what's meant by the term.  Just define it so we have a shared understanding on the term as applied to this process.